radixdlt/src/main/java/com/radixdlt/consensus/View.java

```diff
@@ -58,7 +58,7 @@ public boolean equals(Object o) {
 
 	@Override
 	public String toString() {
-		return "View " + this.view;
+		return Long.toString(this.view);
```

Comment:
How does this class *still* have no header comment?

Also, String.valueOf is more common than Long.toString, but doesn't really matter.